packages/central-server/src/tests/apiV2/import/importSurveyResponses/testGeneral.js

```diff
@@ -29,8 +29,14 @@ export const testGeneral = async () => {
     await app.grantFullAccess();
 
     await findOrCreateDummyCountryEntity(models, { code: 'DL', name: 'Demo Land' });
-    const entities = ['DL_1', 'DL_5', 'DL_7', 'DL_9'].map(code => ({
-      code,
+    const entities = [
+      { code: 'DL_1', name: 'Port Douglas' },
+      { code: 'DL_5', name: 'Hawthorn East' },
+      { code: 'DL_7', name: 'Lake Charm' },
+      { code: 'DL_9', name: 'Thornbury' },
+    ].map(entity => ({
+      code: entity.code,
+      name: entity.name,
       country_code: 'DL',
     }));
     await findOrCreateRecords(models, { entity: entities });
```

Comment:
Is there a test anywhere that explicitly checks the entityId has been updated? Same with the other tests you modified?